blocks/columns/columns.js

```diff
@@ -26,6 +26,18 @@ export default function decorate(block) {
       const pic = col.querySelector('picture');
       const text = col.querySelector('p');
 
+      // Adds Href to image if linked-images class was set
+      if (pic && text && block.classList.contains('linked-images')) {
+        const a = block.querySelector('a');
```

Comment:
should we have here ```col..querySelector('a')``` instead of ```block..querySelector('a')```

Reply:
Sorry saw the comment to late. I will follow up on it here: #364